tests/test_teamcity.py

```diff
@@ -0,0 +1,89 @@
+# stdlib
+import unittest
+
+# 3p
+from mock import MagicMock, patch
+
+# project
+from tests.common import load_check
+
+CONFIG = {
+    'init_config': {'server': 'localhost:8111'},
```

Comment:
What if you have multiple servers in one host ? 
It seems to me that should be part of the instance configuration, not the init_config